modules/smartxBidAdapter.js

```diff
@@ -40,18 +46,29 @@ export const spec = {
       utils.logError(BIDDER_CODE + ': siteId is not present in bidder params');
       return false;
     }
-    if (utils.deepAccess(bid, 'mediaTypes.video.context') == 'outstream' || utils.deepAccess(bid, 'params.ad_unit') == 'outstream') {
+    if (!utils.getBidIdParameter('bidfloor', bid.params)) {
+      utils.logError(BIDDER_CODE + ': bidfloor is not present in bidder params');
+      return false;
+    }
+    if (!utils.getBidIdParameter('bidfloorcur', bid.params)) {
+      utils.logError(BIDDER_CODE + ': bidfloorcur is not present in bidder params');
+      return false;
+    }
+    if (utils.deepAccess(bid, 'mediaTypes.video.context') == 'outstream') {
       if (!utils.getBidIdParameter('outstream_function', bid.params)) {
         if (!utils.getBidIdParameter('outstream_options', bid.params)) {
           utils.logMessage(BIDDER_CODE + ': outstream_options parameter is not defined. The default outstream renderer will be injected in the header. You can override the default SmartX outstream rendering by defining your own Outstream function using field outstream_function.');
-          return true;
+          // return true;
+          return false;
```

Comment:
Was this intended?  If so, can you please update the wording of the previous log message and change it to a warning?  The message above implies that while the setup may not be ideal, it'll still work.  But by returning `false` the bid will be rejected and won't participate in the auction.

Reply:
If 'mediaTypes.video.context' = 'outstream' is set, we need at least outstream_options for our outstream player to initialize. I changed the message to Error and adapted the queries again in the next commit.

Reply:
If 'mediaTypes.video.context' = 'outstream' is set, we need at least outstream_options for our outstream player to initialize. I changed the message to Error and adapted the queries again in the next commit.